src/status_im/chat/models/link_preview.cljs

```diff
@@ -49,8 +49,10 @@
   {:events [::community-resolved]}
   [{:keys [db] :as cofx} community-id community]
   (fx/merge cofx
-            {:db       (community-resolved db community-id)
-             :dispatch [::cache-link-preview-data (community-link community-id) community]}
+            (merge
```

Comment:
I think you can use `assoc` / `cond->`
```
(cond-> {:db (community-resolved db community-id)}
     (nil? community) (assoc :dispatch [::...
```
but `merge` works as well (it just generally signals that you don't know exactly what the value of the second argument is)